.github/workflows/release-fail-bot.yml

```diff
@@ -0,0 +1,42 @@
+name: Notify Slack user on “Package & Release” failure
+
+on:
+  workflow_run:
+    workflows: ["Package & Release"]
+    types:
+      - completed
+
+jobs:
+  notify_failure:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    steps:
+    - name: Generate Slack Payload
+      id: generate_payload
+      uses: actions/github-script@v4
+      with:
+        script: |
+          const github_ids = [];
+          const slack_mapping = JSON.parse(process.env.SLACK_MAPPING);
+          const repo_name = "${{ github.event.workflow_run.repository.full_name }}";
+          const run_id = ${{ github.event.workflow_run.id }};
+          const run_url = `https://github.com/${repo_name}/actions/runs/${run_id}`;
+          const slack_ids = github_ids.map(id => slack_mapping[id]).filter(id => id); // map GitHub IDs to Slack IDs and remove any undefined values
+          const authors = slack_ids.length ? slack_ids.map(id => `<@${id}>`).join(", ") : github_ids.join(", ");
+          const workflow_name = "${{ github.event.workflow_run.name }}"
+          const payload = {
```

Comment:
for this notification we care less about the author on the commit it runs, because builds are daily (or every 3 days) so the author of latest commit is not necessary the one that broke the build. so let's do the following:
- revmoe the author name and mapping
- let's also open a EE sister PR, the workflow is slightly different than that of CE; let's print the workflow parameter (branch and input.version) on the EE side of PR

Reply:
ok